e2e/commands/add.e2e.js

```diff
@@ -685,4 +687,66 @@ describe('bit add command', function () {
       expect(bitMap).not.to.have.property('bar/foo');
     });
   });
+
+  describe('add component/s with gitignore', () => {
+    let errorMessage;
+    before(() => {
+      helper.reInitLocalScope();
+    });
+    it('Should show warning msg in case there are no files to add beacuse of gitignore', () => {
+      helper.createComponent('bar', 'foo2.js');
+      helper.writeGitIgnore(['bar/foo2.js']);
+
+      try {
+        helper.addComponent(path.normalize('bar/foo2.js'));
+      } catch (err) {
+        errorMessage = err.message;
+      }
+      expect(errorMessage).to.contain('warning: no files to add,the following files were ignored: bar/foo2.js');
+    });
+    it('Should show warning msg in case there are no files to add beacuse of gitignore', () => {
```

Comment:
this is the same description as the previous test case.
change it. (otherwise if it will fail we won't know which of them failed)